test/aria/widgets/form/BaseDefaultErrorMessagesTest.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2015 Amadeus s.a.s.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Aria.classDefinition({
+    $classpath : "test.aria.widgets.form.BaseDefaultErrorMessagesTest",
+    $extends : "aria.jsunit.TemplateTestCase",
+    $dependencies : ["aria.utils.Array", "aria.utils.Json"],
+    $constructor : function (widgetName, messagesList) {
```

Comment:
I would add just 2 lines to briefly explain what are supposed to do new tests that extend this one (or modify the ones already implemented) and what they should define.